test/Datadog.Trace.ClrProfiler.Managed.Tests/ScopeFactoryTests.cs

```diff
@@ -13,6 +19,13 @@ public class ScopeFactoryTests
         // declare here instead of using ScopeFactory.UrlIdPlaceholder so tests fails if value changes
         private const string Id = "?";
 
+        public static IEnumerable<object[]> GetDbCommandScopeData()
+        {
+            yield return new object[] { (Func<Tracer, Scope>)SqlCommandCreateScope, typeof(SqlCommand).FullName };
```

Comment:
We probably want to add both `System...SqlCommand` and `Microsoft...SqlCommand`.

Reply:
Good idea, done in dd182a43ad319ccde5254004faa2639044fbb6c7